draft-ietf-taps-interface.md

```diff
@@ -3408,7 +3444,7 @@ Connection -> ConnectionError<reason?>
 ~~~
 
 
-# Connection State and Ordering of Operations and Events
+# Connection State and Ordering of Operations and Events {#state-ordering}
```

Comment:
For context I read the text below this modification. It is stated that 

> Each transition of Connection state is associated with one of more events:

But this does not seem correct, there is no event associated with the transition to Closing described in the text or Figure. Searching the doc for references to Closing, it does not seem to be used?

Reply:
Thanks for reading this!  I must admit, I didn't check - there seems to be a discrepancy here, to be fixed!